### PR TITLE
V3 deploy changes

### PR DESCRIPTION
**Pull request by darkodosenovic**
🐘 

Changing deploy process to be able to deploy V3 branch separately
Adjusting test
Adding manifest class for clearing files
Cleaning variables we don't need

REF: https://amaabca.visualstudio.com/membership_backlog/_workitems/edit/18206

### CODE REVIEW

File: lib/ama/styles/internal/base.rb [hunk 1]
```diff
@@ -18,5 +18,7 @@
 require_relative 'environment'
 require_relative 'chat'
 require_relative 'cli'
+require_relative 'manifest'
```
Comment (mvandenbeuken) [inline lib/ama/styles/internal/base.rb]: double requirement for manifest

File: lib/ama/styles/internal/deployment.rb [hunk 2]
```diff
@@ -17,8 +17,9 @@ def run
           upload_files
           log('Generating fallback stylesheet...'.colorize(:yellow))
           upload_fallback_stylesheet
+          upload_latest_stylesheet
           log('Verifying S3 integrity...'.colorize(:yellow))
-          request
+          collect_garbage!
```
Comment (mvandenbeuken) [inline lib/ama/styles/internal/deployment.rb]: collect that garbage!

File: lib/ama/styles/internal/environment.rb [hunk 3]
```diff
@@ -40,18 +40,15 @@ def configure_redis
...
+          config.assets_bucket_name = ENV['V3_S3_ASSETS_BUCKET_NAME']
         end
 
         def configure_aws_client
           Aws.config.update(
             region: 'us-west-2',
-            credentials: Aws::Credentials.new(
-              config.aws_access_key_id,
-              config.aws_secret_access_key
+            credentials: Aws::SharedCredentials.new(
```
Comment (mvandenbeuken) [inline lib/ama/styles/internal/environment.rb]: this is a good change

**Comment by darkodosenovic** [comment]
:wave:

**Pull request merged**